source/manual/emergency-publishing.html.md

```diff
@@ -1,162 +0,0 @@
----
```

Comment:
You could set up a redirect for this file here: https://github.com/alphagov/govuk-developer-docs/blob/master/data/redirects.yml